libraries/adaptive-expressions/package.json

```diff
@@ -24,7 +24,7 @@
     "@types/lru-cache": "^5.1.0",
     "@types/moment-timezone": "^0.5.13",
     "@types/xmldom": "^0.1.29",
-    "antlr4ts": "0.5.0-alpha.3",
```

Comment:
it  is better to fix the version of antlr, because every small change is breaking change for the Lexer/Parser

Reply:
ok，fixed